docs/src/release-notes-js.md

```diff
@@ -96,12 +96,12 @@ Read more about [component testing with Playwright](./test-components).
     }
   });
   ```
-* If you intend to edit HAR by hand, consider using the `"minimal"` HAR recording mode 
+* If you intend to edit HAR by hand, consider using the `"minimal"` HAR recording mode
   that only records information that is essential for replaying:
   ```ts
   const context = await browser.newContext({
     recordHar: {
-      path: 'github.har.zip',
+      path: 'github.har',
```

Comment:
I made this change since this snippet is for `If you intend to edit HAR by hand`, so I think it doesn't make sense to have .zip. Is that correct?

Reply:
Yeah, makes sense